mscore/pagesettings.cpp

```diff
@@ -189,65 +197,47 @@ void PageSettings::updateValues()
                   }
             }
       pageGroup->setCurrentIndex(index);
-//      QString s;
-
-      //qreal printableWidthValue = pf->printableWidth();
-      qreal widthValue = pf->size().width();
-      if (mm) {
-            oddPageTopMargin->setValue(pf->oddTopMargin() * INCH);
-            oddPageBottomMargin->setValue(pf->oddBottomMargin() * INCH);
-            oddPageLeftMargin->setValue(pf->oddLeftMargin() * INCH);
-            oddPageRightMargin->setValue(pf->oddRightMargin() * INCH);
-
-            evenPageTopMargin->setValue(pf->evenTopMargin() * INCH);
-            evenPageBottomMargin->setValue(pf->evenBottomMargin() * INCH);
-            evenPageLeftMargin->setValue(pf->evenLeftMargin() * INCH);
-            evenPageRightMargin->setValue(pf->evenRightMargin() * INCH);
-
-            spatiumEntry->setValue(sc->spatium()/DPMM);
-            pageHeight->setValue(pf->size().height() * INCH);
-            widthValue          *= INCH;
-            }
-      else {
-            oddPageTopMargin->setValue(pf->oddTopMargin());
-            oddPageBottomMargin->setValue(pf->oddBottomMargin());
-            oddPageLeftMargin->setValue(pf->oddLeftMargin());
-            oddPageRightMargin->setValue(pf->oddRightMargin());
-
-            evenPageTopMargin->setValue(pf->evenTopMargin());
-            evenPageBottomMargin->setValue(pf->evenBottomMargin());
-            evenPageLeftMargin->setValue(pf->evenLeftMargin());
-            evenPageRightMargin->setValue(pf->evenRightMargin());
-
-            spatiumEntry->setValue(sc->spatium()/DPI);
-            pageHeight->setValue(pf->size().height());
-            }
-      pageWidth->setValue(widthValue);
 
+      spatiumEntry->setValue(sc->spatium() / converter);
+
+      qreal widthValue = pf->size().width() / converter;
+      pageWidth->setValue(widthValue);
       oddPageLeftMargin->setMaximum(widthValue);
       oddPageRightMargin->setMaximum(widthValue);
       evenPageLeftMargin->setMaximum(widthValue);
       evenPageRightMargin->setMaximum(widthValue);
 
-      evenPageTopMargin->setEnabled(pf->twosided());
-      evenPageBottomMargin->setEnabled(pf->twosided());
-      evenPageLeftMargin->setEnabled(pf->twosided());
-      evenPageRightMargin->setEnabled(pf->twosided());
-
-      if (twosided->isChecked()) {
-            evenPageRightMargin->setValue(oddPageLeftMargin->value());
-            evenPageLeftMargin->setValue(oddPageRightMargin->value());
+      // ???Why no setMaximum for height???
```

Comment:
The code above this line note sets max values for the left and right margins.  There is no code to set max values for the top or bottom margins.  Is there a reason for that, or should it be added?